djdt_flamegraph/djdt_flamegraph.py

```diff
@@ -51,20 +46,24 @@ def enabled(self):
 
     @property
     def content(self):
-        ctx = {
+        return Template(template).render(Context({
             'flamegraph': flamegraph.stats_to_svg(self.sampler.get_stats())
-        }
-        return Template(template).render(Context(ctx))
```

Comment:
This doesnt appear to have changed at all?  Why modify it?

Reply:
It's a leftover from other attempts to fix the scripts not executing that i ended up removing.
I can remove it?

Reply:
Restoring this chunk would be good.

Also I think the minimum supported version of djdt should be increased to 2.2 - that is what works for me.

Reply:
ping @aleksanb , can you restore this, so that the changes needed are more clear and not distorted by unneeded changes.